### ISSUE TITLE
[bt4g] (testing) Exception (bt4g): FlareSolverr was unable to process the request, please check FlareSolverr logs. 

### ISSUE THREAD

**Issue by davidreverett** [FlareSolverr/FlareSolverr#562]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

### How to enable debug and html traces

[Follow the instructions from this wiki page](https://github.com/FlareSolverr/FlareSolverr/wiki/How-to-enable-debug-and-html-trace)

### Environment

* **FlareSolverr version**:2.2.10
* **Last working FlareSolverr version**:2.2.9
* **Operating system**: Windows 10
* **Are you using Docker**: [yes/no] No
* **FlareSolverr User-Agent (see log traces or / endpoint)**: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:94.0) Gecko/20100101 Firefox/94.0
* **Are you using a proxy or VPN?** [yes/no] Yes
* **Are you using Captcha Solver:** [yes/no] No
* **If using captcha solver, which one:** N/A
* **URL to test this issue:** https://bt4g.org/search/2022

### Description

Installed 2.2.10 to see if it addressed some of the recent issues- still seeing problems with bt4g- was working up until recently when several search sites broke with the same errors due to known issues. 

Steps to reproduce: 

1. Click test site button in Jackett for BT4g  with 2.2.10 configured and working for other sites. Jackett returns the following error: 

### Logged Error Messages

Jackett:

Exception (bt4g): FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Unable to process browser request. ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5 UnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1 navigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13 execute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25 execute@chrome://remote/content/cdp/sessions/Session.jsm:64:25 execute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20 onPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36 onMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18 handleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14 : FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Unable to process browser request. ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5 UnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1 navigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13 execute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25 execute@chrome://remote/content/cdp/sessions/Session.jsm:64:25 execute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20 onPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36 onMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18 handleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14

Flaresolverr Debug logs:

C:\Program Files\flaresolverr>set LOG_LEVEL=debug

C:\Program Files\flaresolverr>set LOG_HTML=true

C:\Program Files\flaresolverr>flaresolverr.exe
2022-10-22T11:16:00-07:00 INFO FlareSolverr v2.2.10
2022-10-22T11:16:00-07:00 DEBUG Debug log enabled
2022-10-22T11:16:00-07:00 INFO Testing web browser installation...
2022-10-22T11:16:00-07:00 DEBUG FlareSolverr user home directory is OK: C:\Users\Plex-NUC
2022-10-22T11:16:00-07:00 DEBUG Test URL: https://www.google.com
2022-10-22T11:16:00-07:00 DEBUG Creating new session...
2022-10-22T11:16:00-07:00 DEBUG Launching web browser...
2022-10-22T11:16:02-07:00 INFO FlareSolverr User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:94.0) Gecko/20100101 Firefox/94.0
2022-10-22T11:16:03-07:00 INFO Test successful
2022-10-22T11:16:03-07:00 INFO Listening on http://0.0.0.0:8191
2022-10-22T11:16:26-07:00 INFO REQ-1 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://bt4g.org/search/2022","proxy":{"url":"192.168.0.187:8118"}}
2022-10-22T11:16:26-07:00 DEBUG REQ-1 Creating new session...
2022-10-22T11:16:26-07:00 DEBUG REQ-1 Using proxy: 192.168.0.187:8118
2022-10-22T11:16:26-07:00 DEBUG REQ-1 Launching web browser...
2022-10-22T11:16:27-07:00 DEBUG REQ-1 Using proxy: 192.168.0.187:8118
2022-10-22T11:16:27-07:00 DEBUG REQ-1 Navigating to... https://bt4g.org/search/2022
2022-10-22T11:16:46-07:00 ERROR REQ-1 Unexpected error: ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5
UnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1
navigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13
execute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25
execute@chrome://remote/content/cdp/sessions/Session.jsm:64:25
execute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20
onPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36
onMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18
handleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14

2022-10-22T11:16:46-07:00 ERROR REQ-1 Error: Unable to process browser request. ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5
UnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1
navigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13
execute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25
execute@chrome://remote/content/cdp/sessions/Session.jsm:64:25
execute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20
onPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36
onMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18
handleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14

2022-10-22T11:16:46-07:00 INFO REQ-1 Response in 20.077 s

**Comment by davidreverett** [comment]
Seems this may already be logged as #536 but was supposed to be fixed in 2.2.10? It lists several other sites that are also still not working. So my issue might be a duplicate....

**Comment by ilike2burnthing** [comment]
Yes, this is a duplicate of https://github.com/FlareSolverr/FlareSolverr/issues/536.

No, v2.2.10 was not supposed to fix it, we'll likely be waiting on v3 for that, which is currently in beta.
